a10_octavia/common/a10constants.py

```diff
@@ -93,8 +93,13 @@
 CONFIRM_VRRP_STATUS = 'confirm-vrrp-status'
 ALLOCATE_VIP = 'allocate-vip'
 UPDATE_VIP_AFTER_ALLOCATION = 'update-vip-after-allocation'
+AMPHORAE_POST_VIP_PLUG = 'amphorae-post-vip-plug'
 CREATE_VTHUNDER_ENTRY = 'create-vthunder-entry'
+UPDATE_ACOS_VERSION_IN_VTHUNDER_ENTRY = 'update-acos-version-in-vthunder-entry'
 VTHUNDER_BY_LB = 'vthunder-by-loadbalancer'
+GET_VTHUNDER_BY_LB = "get-vthunder-by-lb"
+VTHUNDER_CONNECTIVITY_WAIT = "vThunder-connectivity-wait"
```

Comment:
A nitpick, but the "T" here needs to be lowercase too keep consistency with the other constants

Reply:
Done